### DISCUSSION TITLE
Adding SEI parameters to available parameter set

### DISCUSSION THREAD

**Discussion by zafar594** [pybamm-team/PyBaMM#4007] [Q&A]
Hi! I am trying to add SEI parameters to already existing parameter set Prada2013. Adding parameters individually is causing error ("can't update cos it doesn't exist", though I'm trying to add instead of update). So I saved the parameter file locally, edited it to add parameters that I want. Now, how do I use "get_paramete_values" function to use my own parameter file stored locally? Please help!

**Comment by valentinsulzer** [comment]
- to add new parameters just add the flag check_already_exists=False
- to add a new parameter set so that you can call it as a string e.g.ParameterValues("Chen2020") you have to set it up as an entry point: https://docs.pybamm.org/en/stable/source/api/parameters/parameter_sets.html#adding-parameter-sets
